### ISSUE TITLE
自行编译安卓版本在 vivopad2 上无法运行

### ISSUE THREAD

**Issue by hac425xxx** [siyuan-note/siyuan#12224]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Can the issue be reproduced with the default theme (daylight/midnight)?

- [X] I was able to reproduce the issue with the default theme

### Could the issue be due to extensions?

- [X] I've ruled out the possibility that the extension is causing the problem.

### Describe the problem

自己编译的安卓应用在小米，模拟器上都能运行，但是vivopad上就不行，在f-droid上下的版本也是这样，但是vivo应用商店下载的就能正常运行

f-droid 版本下载链接
https://f-droid.org/en/packages/org.b3log.siyuan/

报错截图

![A}_ @Q5V)F3$K BT~U2O5FI](https://github.com/user-attachments/assets/32711e91-2d84-4799-8aa1-565af395b65b)

我尝试把6806端口换成 26806  也不行，感觉应该不是端口权限的问题

但是从 vivo 应用市场下载的应用能够正常运行，请问如何编译能在 vivopad 上运行的应用

### Expected result

正常运行

### Screenshot or screen recording presentation

x

### Version environment

```markdown
- Version: 3.1.3
- Operating System: 
- Browser (if used):
```

### Log file

x

### More information

x

**Comment by 88250** [comment]
这个我不太清楚原因，建议试下手动打开所有权限。

**Comment by ShibuyaCyana** [comment]
Pretty much the same problem. Everything works fine for the F-droid version, except the **local network servicing**.
This auth page appears as expected - and it would block my access if I entered a wrong password, but with the right password it then shows up a 404 error.

![image](https://github.com/user-attachments/assets/d15c3dde-0731-4d79-8e62-ddf09fc464fd)

![image](https://github.com/user-attachments/assets/5114a6e3-036a-443e-9993-abc82dfd376c)

This function works correctly If I use the version downloaded here: https://b3log.org/siyuan/en/download.html, But this link is buried deeply in the help document.
Also I noticed that the F-droid apk size is smaller than the one downloaded from b3blog.org (37.7 MB vs 45.9MB respectively).

SiYuan apk: 3.1.11
Browser: Edge
